Makefile

```diff
@@ -121,7 +132,7 @@ ifeq (, $(shell which controller-gen))
 	CONTROLLER_GEN_TMP_DIR=$$(mktemp -d) ;\
 	cd $$CONTROLLER_GEN_TMP_DIR ;\
 	go mod init tmp ;\
-	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0 ;\
+	go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1 ;\
```

Comment:
Idea: we could use a `generate.go` like we did in the lieutenant and trigger the generation through that. https://github.com/projectsyn/lieutenant-operator/blob/master/generate.go

Pros: the version of controller-gen can be managed at one place via the `go.mod` file. Also it would automatically go get the right version if not available.

Reply:
That's interesting, I'll try it out!